ClientSideEncryptionToServerSideEncryptionMigrationSamples/ClientSideKeyVaultKeyToCustomerProvidedKey/key_wrapper.py

```diff
@@ -0,0 +1,30 @@
+
+from azure.keyvault.keys.crypto import CryptographyClient
+from settings import *
+
+class KeyWrapper:
+    # key wrap algorithm for kek
```

Comment:
Super mega nit: capital 'K' since every other comment you've had up to this point of the review uses capital letters in the first word of the comments

Reply:
Just deleted the comment since it didn't make any sense :)